app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/conversationListItem.tsx

```
@@ -33,8 +36,85 @@ export const ConversationListItem = ({
 }: ConversationListItemProps) => {
   const listItemRef = useRef<HTMLAnchorElement>(null);
   const { mailboxSlug } = useConversationListContext();
+  const { category } = useParams<{ category: string }>();
```

ESLint

[error] 'category' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)